ui/dist/asset-manifest.json

```diff
@@ -1,8 +1,8 @@
 {
-  "app.css": "/static/app.d42031fb.css",
-  "app.js": "/static/app-65d69047.js",
-  "app.css.map": "/static/app.d42031fb.css.map",
-  "app.js.map": "/static/app-65d69047.js.map",
+  "app.css": "/static/app.10bd0b1e.css",
```

Comment:
why are the static assets different from dev? That looks like a merge issue to me and should be reset